app/src/main/java/ch/epfl/qedit/view/answer/MatrixFragment.java

```diff
@@ -0,0 +1,107 @@
+package ch.epfl.qedit.view.answer;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.text.InputFilter;
+import android.text.InputType;
+import android.text.method.DigitsKeyListener;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.WindowManager;
+import android.widget.EditText;
+import android.widget.TableLayout;
+import android.widget.TableRow;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import ch.epfl.qedit.R;
+import ch.epfl.qedit.model.MatrixFormat;
+import java.util.ArrayList;
+
+public class MatrixFragment extends Fragment {
+    private TableLayout tableLayout;
+    private boolean hasDecimal = true;
+    private boolean hasSign = true;
+
+    private ArrayList<TableRow> tableRow = new ArrayList<>();
+    private ArrayList<ArrayList<EditText>> arrayButtons = new ArrayList<>();;
```

Comment:
Typo

Reply:
Typo on what ? I may be blind but I really don't see the typo

Reply:
Two semicolons, that’s it ;)